doc/manuals/admin/extra/indexes_analysis.md

```diff
@@ -0,0 +1,73 @@
+# Database indexes analysis
+
+The following analyzis shows the TPS (transation per second) and storage
+consumption figures for different indexes configuration and number of
```

Comment:
... figures for configurations with different indexes and different numbers of entities ...